PinvokeAnalyzer_UWPApis.txt

```diff
@@ -0,0 +1,2298 @@
+bcrypt.dll!BCryptCloseAlgorithmProvider
```

Comment:
In essence, yes. The actual algorithm is:
* Take the existing API list, filter out all functions that are mapped to API sets, and keep functions that are mapped to normal DLL names.
* Dumpbin is used to dump the contents of common import libraries in Windows SDK, such as `kernel32.lib`. The output is then parsed to extract function names and their associated DLL names. (The list generated by this should be more stable than function names directly extracted from DLLs.)
* Function names extracted from the import libraries are added to the API list only if the function already existed in the original API list but mapped to an API set. This prevents unnecessarily expanding the size of the API list.
* Three API set names are treated specially. `api-ms-win-core-winrt-error-l1-1-0.dll`, `api-ms-win-core-memory-l1-1-3.dll`, and `api-ms-win-core-winrt-robuffer-l1-1-0.dll` are still used by corefx, and no equivalent traditional DLL names are available. Thus functions from these three API sets are copied as-is from the original API list to the new one.